internal/api/grpc_service.go

```diff
@@ -27,3 +28,121 @@ func (s *GrpcService) GetDeviceDefinitionByID(ctx context.Context, in *p_grpc.Ge
 
 	return result, nil
 }
+
+func (s *GrpcService) GetDeviceDefinitionByMMY(ctx context.Context, in *p_grpc.GetDeviceDefinitionByMMYRequest) (*p_grpc.GetDeviceDefinitionItemResponse, error) {
+
+	qryResult, _ := s.Mediator.Send(ctx, &queries.GetDeviceDefinitionByMakeModelYearQuery{
+		Make:  in.Make,
+		Model: in.Model,
+		Year:  int(in.Year),
+	})
+
+	dd := qryResult.(queries.GetDeviceDefinitionByMakeModelYearQueryResult)
+
+	result := &p_grpc.GetDeviceDefinitionItemResponse{
+		DeviceDefinitionId: dd.DeviceDefinitionID,
+		Name:               dd.Name,
+		ImageUrl:           dd.ImageURL,
+		Type: &p_grpc.GetDeviceDefinitionItemResponse_Type{
+			Type:  dd.Type.Type,
+			Make:  dd.Type.Make,
+			Model: dd.Type.Model,
+			Year:  uint32(dd.Type.Year),
+		},
+		VehicleData: &p_grpc.GetDeviceDefinitionItemResponse_VehicleInfo{
+			FuelType:            dd.VehicleInfo.FuelType,
+			DrivenWheels:        dd.VehicleInfo.DrivenWheels,
+			NumberOfDoors:       dd.VehicleInfo.NumberOfDoors,
+			Base_MSRP:           uint32(dd.VehicleInfo.BaseMSRP),
+			EPAClass:            dd.VehicleInfo.EPAClass,
+			VehicleType:         dd.VehicleInfo.VehicleType,
+			MPGHighway:          dd.VehicleInfo.MPGHighway,
```

Comment:
Let's also add MPG - we currently don't have it but I had recently added it to `devices-api` https://github.com/DIMO-Network/devices-api/blob/498fdda84e6c9c8b296594daff936a951e451a60/internal/services/models.go#L66